python/vowpalwabbit/pyvw.py

```diff
@@ -1262,8 +1262,8 @@ class CBEvalLabel(AbstractLabel):
 
     def __init__(
         self,
-        action: int = 0,
-        cb_label: CBLabel = CBLabel(),
+        action: Optional[int] = None,
+        cb_label: Optional[CBLabel] = None,
     ):
```

Comment:
I'd really rather not have optional fields. If youre using this type they need to be set

Reply:
removed